starter-project-web-vue2/services/ArticleService.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
```

Comment:
we could have used $axios here since it's already set up with nuxt. If the user is logged in, it will automatically attach the tokens with a normal request like $axios.get(), etc. 


I dont think it matters now tho.